src/modules/Storage.js

```diff
@@ -0,0 +1,7 @@
+const getTasks = () => JSON.parse(localStorage.getItem('tasks')) || [];
+
+const saveTasks = (tasksArray) => {
+  localStorage.setItem('tasks', JSON.stringify(tasksArray));
+};
```

Comment:
Great design @andrianarivo 🤝

- [ ] As stated in the project requirements, kindly ensure the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). This is not the case for your design. Kindly work on this and submit your project for approval.
![Indexes](https://user-images.githubusercontent.com/63917725/235837149-51f2e663-a2b2-4703-8eca-bb21227f89cd.PNG)

Reply:
@1GORDON I need more clarifications, are you able to join a slack call ?